components/script/dom/htmlformelement.rs

```diff
@@ -258,8 +262,83 @@ impl HTMLFormElementMethods for HTMLFormElement {
 
     // https://html.spec.whatwg.org/multipage/#the-form-element%3Adetermine-the-value-of-a-named-property
     fn NamedGetter(&self, name: DOMString) -> Option<RadioNodeListOrElement> {
-        // return Option::Some::<RadioNodeListOrElement>;
-        unimplemented!();
+        // unimplemented!();
+        let window = window_from_node(self);
+
+        // Q2. ERROR in the following declaration - check the attached screenshot (image2)
+        let mut candidates: Vec<Dom<HTMLElement>> = Vec::new();
+
+        // let mut candidates: RadioNodeList = RadioNodeList::new(&window, Dom<HTMLElement>);
+
+        let controls = self.controls.borrow();
+
+        for child in controls.iter() {
+            if child
+                .downcast::<HTMLElement>()
+                .map_or(false, |c| c.is_listed_element())
+            {
+                if child.has_attribute(&local_name!("id")) ||
+                    (child.has_attribute(&local_name!("name")) &&
+                        child.get_string_attribute(&local_name!("name")) == name)
+                {
+                    // println!("{:?}", child.what_is_this());
+                    candidates.push(Dom::from_ref(&*child.downcast::<HTMLElement>().unwrap()));
```

Comment:
This will need to use `child.upcast::<Node>()` instead.

Reply:
> No screenshots were attached anywhere that I could find.

Image 1 for Q1:

![image1](https://user-images.githubusercontent.com/16729243/70269215-edffe280-176f-11ea-8a0e-72dcb7227674.png)

Reply:
I think you need to pass into_iter() instead of iter() as the second argument to RadioNodeList::new_simple_list.

Reply:
Fixed.